src/main/java/hudson/remoting/Request.java

```diff
@@ -341,7 +341,7 @@ public RSP get(long timeout, TimeUnit unit) throws InterruptedException, Executi
      * Aborts the processing. The calling thread will receive an exception.
      */
     /*package*/ void abort(IOException e) {
-        onCompleted(new Response(this, id, 0, new RequestAbortedException(e)));
+        onCompleted(new Response<>(this, id, 0, new RequestAbortedException(e)));
```

Comment:
This change would not have worked if we had not expanded the bound above to be able to include `RequestAbortedException`.